### ISSUE TITLE
syriac - enable linked definition credits

### ISSUE THREAD

**Issue by balmas** [alpheios-project/alpheios-core#171]
for sources, George would like to try something like the following (modeled after the ge'ez approach)

1. SyriacWord1 [SedraLink]
       part-of-speech
       gloss1, gloss2, gloss3 (Dictionary1Abbreviation); gloss4, gloss5, gloss6 (Dictionary2Abbreviation)
2. ditto

Credits
 Sedra [with a link to sedra.bethmardutho.org], Georg A. Kiraz (General Editor), James W. Bennet (IT Manager)
 Dictionary1Abbreiviaiton = <full citation>
 Dictionary2Abbreivation = <full citation>
(but don't duplicate if an abbreviation appears more than once and list them in alphabetical order of the abbreviations)

_Originally posted by @balmas in https://github.com/alpheios-project/alpheios-core/issues/162#issuecomment-589675162_

**Comment by balmas** [comment]
@irina060981 can you give this a try? You can take a look at what we do in the Traces engine for the link to Sedra.  

I was thinking that we can move the wrapping of the definition in the ResourceProviderProxy to the engine-specific parseMeaning function.

Let me know if you have trouble finding enough Syriac words to test and develop with. I had added one fixture but we probably need more.   You can use the Sedra lookup tool at http://sedra.bethmardutho.org/lexeme/get and the keyboard they have there to get some words using the autocomplete

**Comment by irina060981** [comment]
@balmas, I have examined the way that we could add definitions.
We have separate definitions for Greek, for example

![image](https://user-images.githubusercontent.com/12377640/76517907-e86e1300-64a9-11ea-99d1-88112dd909c8.png)

But we have the main difference here.
For Greek we currently none - what exactly provider we have - each source (provider) has its own config data in config json, and each definition is connected to one source.

But here we have the other structure (I believe).
For example for the given word - the first lexicon - ܘܐܡܪܝܢ
https://morph.alpheios.net/api/v1/analysis/word?word=%DC%98%DC%90%DC%A1%DC%AA%DC%9D%DC%A2&engine=sedra&lang=syr&clientId=embed-lib-index-demo

we have the following short definitions:
![image](https://user-images.githubusercontent.com/12377640/76518829-8d3d2000-64ab-11ea-9b0c-061cdef18104.png)

and separately the following source data
![image](https://user-images.githubusercontent.com/12377640/76518862-9cbc6900-64ab-11ea-9a51-3c4a68f55dbc.png)

and that's why we have the following problems:
1. the gloss and the dictionary have no mapping for each other
2. we have only full name of the source, and don't have 
     - short dictionary name to link to the gloss
     - rightsUri to define the dictionary inside ResourceProvider

but, if we could solve this moments, we could do the requested structure, I think.

@balmas , what do you think?

**Comment by balmas** [comment]
yes you're right we don't have a way to tie the individual meanings to their sources. I'll check with the Sedra team to see if they want to make that change to enable a link between the meanings and the sources, or if they just want us to aggregate and dedupe them. Stay tuned :-)

**Comment by balmas** [comment]
taking this one back for now
